### DISCUSSION TITLE
Thresholding Syntax

### DISCUSSION THREAD

**Discussion by Cra5hedC0w** [Security-Onion-Solutions/securityonion#2639] [Unsupported Versions] [answered]
I am attempting to suppress certain alerts based on criteria, such as src/dst. I know there is a snip of information for syntax for thresholding on the SO docs. Is all of that data required, or would something such as this work to suppress all SIDs generated containing the src/dst as specified: 

thresholding:
  sids:
    201630:
      - suppress:
          gen_id: 1
          track: by_src
          ip: 10.10.10.1/24
    2101411:
      - suppress:
          gen_id: 1
          track: by_dst
          ip: 10.10.10.2/24

When I run the update to apply idstools, there are no complaints, however I am still seeing these SIDs triggering alerts for the hosts in question.  (Excuse the spacing, for some reason my post won't hold the indentations)

**Comment by greatapoc** [comment, marked as answer]
The /24 is unnecessary and that's probably why it's not suppressing them.

**Reply by Cra5hedC0w** [reply]
Thanks, I made the adjustments, and that appears to work (so far). I will point out, that in the documentation for the syntax examples, there are CIDR notations for entire subnets, but not individual hosts.
